packages/contracts/contracts/L1/L2OutputOracle.sol

```diff
@@ -310,10 +311,9 @@ contract L2OutputOracle is Initializable, Semver {
     }
 
     /**
-     * @notice Returns the number of outputs that have been submitted. Will revert if no outputs
-     *         have been submitted yet.
+     * @notice Returns the index of the latest submitted output.
      *
-     * @return The number of outputs that have been submitted.
```

Comment:
Is there any reason the comment about `revert` was removed? It still seems to revert when `l2Outputs.length` is 0.

Reply:
Oh I see, I will restore the comment. Thx!